development/image-syncer/external-images.yaml

```diff
@@ -48,4 +48,6 @@ images:
   tag: "2.7.4-alpine-2022-04-05"
 - source: "natsio/nats-server-config-reloader:0.6.3"
 - source: "natsio/prometheus-nats-exporter:0.9.1"
```

Comment:
This line can be safely deleted, as the image won't be deleted from our mirror

Reply:
done